<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatFri</class>
 <widget class="QWidget" name="ChatFri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="messBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>731</width>
     <height>231</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="face.qrc">
     <normaloff>:/face/face00.ico</normaloff>:/face/face00.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/ico/tp.png</normaloff>:/ico/tp.png</iconset>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>50</y>
     <width>256</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="messEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>731</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="face.qrc"/>
 </resources>
 <connections/>
</ui>
